### PR TITLE
Add digital-clock and alarm clock

### PR DESCRIPTION
**Pull request by ocditimban**
Add digital-clock and alarm clock

### CODE REVIEW

**Comment by tienvx** [comment]
There are a lot of files that are not belong to the library that is used. e.g.  app/images/background.png

File: app/index.html [hunk 1]
```diff
@@ -39,11 +40,14 @@
     <script src="libraries/bootstrap-select/dist/js/bootstrap-select.js"></script>
     <script src="libraries/coolclock/coolclock.js"></script>
     <script src="libraries/coolclock/moreskins.js"></script>
-    <script src="libraries/jquery-clock-plugin/js/jquery.clock.js"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/moment.js/2.0.0/moment.min.js"></script>
```
Comment (tienvx) [inline app/index.html]: We need to use bower to download moment library:
```
  "dependencies": {
    "moment": "version",
  },
```

File: app/index.html [hunk 2]
```diff
@@ -39,11 +40,14 @@
     <script src="libraries/bootstrap-select/dist/js/bootstrap-select.js"></script>
     <script src="libraries/coolclock/coolclock.js"></script>
     <script src="libraries/coolclock/moreskins.js"></script>
-    <script src="libraries/jquery-clock-plugin/js/jquery.clock.js"></script>
```
Comment (tienvx) [inline app/index.html]: Why this file is deleted? It's used in another directive.

**Comment by andytruong** [comment]
Whoa

**Pull request merged**